wsmaster/codenvy-sql-schema/src/main/resources/codenvy-schema/5.0.0/1.1__init.sql

```diff
@@ -0,0 +1,37 @@
+--
```

Comment:
1. `init.sql` is bad name in this case.
2. I'm not sure that 1.1 is right version here.